packages/core/src/shared/telemetry/util.ts

```diff
@@ -180,15 +187,20 @@ export function getUserAgent(
 type EnvType =
     | 'cloud9'
     | 'cloud9-codecatalyst'
+    | 'cloudDesktop'
```

Comment:
nit: maybe
```suggestion
    | 'amzn-cloudDesktop'
```

Reply:
+1, the nonce "amzn" for all internal stuff is useful.

Reply:
updated